### DISCUSSION TITLE
Button size

### DISCUSSION THREAD

**Discussion by SmartMart69** [FlowiseAI/Flowise#1546] [General]
Hi there,

this has been asked before but unfortunately ignored... ;-)

Any idea how I can get the chatbot button bigger? The sizes "small", "medium" and "large" are not sufficient imho...

Thanks and regards
Martin

**Comment by HenryHengZJ** [comment]
we can definitely look at how to let user specify their own size!
@0xi4o

**Comment by SmartMart69** [comment]
Any news on this one? :-)

**Comment by amansoni7477030** [comment]
Hii @SmartMart69 @HenryHengZJ  I've added functionality to change the icon size of the chatbot based on a custom value. Please check my pull request https://github.com/FlowiseAI/FlowiseChatEmbed/pull/173

i also added other enhancement in that.
